{"name": "Создание заказа с невалидными ингредиентами", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('logged_in_user_credentials', <FixtureRequest for <Function test_create_order_with_invalid_ingredients>>)", "trace": "file /Users/flowforme/stud/Diplom_2/tests/test_create_order.py, line 40\n      @allure.title('Создание заказа с невалидными ингредиентами')\n      @allure.description('Создаём заказ с некорректными форматом хеша ингредиентов, проверяем, что вернулась ошибка 500')\n      def test_create_order_with_invalid_ingredients(self, logged_in_user_credentials):\nE       fixture 'logged_in_user_credentials' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/flowforme/stud/Diplom_2/tests/test_create_order.py:40"}, "description": "Создаём заказ с некорректными форматом хеша ингредиентов, проверяем, что вернулась ошибка 500", "start": 1726571274998, "stop": 1726571274998, "uuid": "b76615d4-8936-4cc1-88d3-bff06cec7987", "historyId": "8da158c03f34cca19354eabf273e4086", "testCaseId": "8da158c03f34cca19354eabf273e4086", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_ingredients", "labels": [{"name": "suite", "value": "Тесты создания заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "MacBook-Pro-Petr.local"}, {"name": "thread", "value": "57416-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}